src/c++/uds/src/uds/permassert.h

```diff
@@ -6,6 +6,10 @@
 #ifndef PERMASSERT_H
 #define PERMASSERT_H
 
+#ifndef __KERNEL__
+#include <linux/build_bug.h>
+#endif
+
```

Comment:
Blank line at line 12 should be removed. We need to make sure not to add extra blank line in the kernel build.